### PR TITLE
Issues in overlay rebuild allowed to continue

### PR DESCRIPTION
**Pull request by wthorp**

### CODE REVIEW

File: pkg/overlay/cache.go [hunk 1]
```diff
@@ -132,19 +129,19 @@ func (o *Cache) Bootstrap(ctx context.Context) error {
...
 		if err != nil {
-			zap.Error(ErrNodeNotFound)
+			zap.L().Info("Node find failed", zap.String("nodeID", v.Id))
+			continue
 		}
-
 		n, err := proto.Marshal(&found)
 		if err != nil {
-			return err
+			zap.L().Error("Node unmarshall failed", zap.String("nodeID", v.Id))
```
Comment (egonelbre) [inline pkg/overlay/cache.go]: should be `Node marshal failed`.

File: pkg/overlay/cache.go [hunk 2]
```diff
@@ -163,21 +160,21 @@ func (o *Cache) Refresh(ctx context.Context) error {
...
 		if err != nil {
-			return err
+			zap.L().Info("Node ping failed", zap.String("nodeID", n.GetId()))
+			continue
 		}
-
 		data, err := proto.Marshal(&pinged)
 		if err != nil {
-			return err
+			zap.L().Error("Node unmarshall failed", zap.String("nodeID", n.GetId()))
```
Comment (egonelbre) [inline pkg/overlay/cache.go]: Should be `Node unmarshal failed`

File: pkg/overlay/cache.go [hunk 3]
```diff
@@ -186,27 +183,24 @@ func (o *Cache) Refresh(ctx context.Context) error {
...
-			zap.Error(ErrNodeNotFound)
-			return err
+			zap.L().Info("Node ping failed", zap.String("nodeID", n.GetId()))
+			continue
 		}
-
 		data, err := proto.Marshal(&pinged)
 		if err != nil {
-			return err
+			zap.L().Error("Node unmarshall failed", zap.String("nodeID", n.GetId()))
```
Comment (egonelbre) [inline pkg/overlay/cache.go]: Should be `Node unmarshal failed`

File: pkg/overlay/cache.go [hunk 4]
```diff
@@ -132,19 +129,19 @@ func (o *Cache) Bootstrap(ctx context.Context) error {
 	for _, v := range nodes {
```
Comment (egonelbre) [inline pkg/overlay/cache.go]: missed one at line 126

Comment (wthorp) [reply pkg/overlay/cache.go]: fixed

File: pkg/overlay/cache.go (hunk 2 above)
Comment (wthorp) [reply pkg/overlay/cache.go]: fixed

**Pull request merged**